client/web/src/insights/components/insight-view-content/InsightViewContent.scss

```
@@ -26,4 +26,11 @@
         width: 100%;
         height: 100%;
     }
+
+    &:hover,
+    &:focus {
```

I think this should be `:focus-within` so it also disappears when e.g. a data point is focused (just like `:hover` triggers if any child element is hovered). The chart `view-content` itself is not focusable, so `:focus` doesn't have any effect. This should fix one half of the problem in my review summary.
```suggestion
    &:focus-within {
```

---

So good! Thx and done.